### DISCUSSION TITLE
Error with intf_tops_parallel.csh

### DISCUSSION THREAD

**Discussion by WY-SVG** [gmtsar/gmtsar#895] [General]
Hello,can you help me please with this error, I tried to apply intf_tops.csh one.in batch_tops.config to run one interferograme with mode = 2 in in batch_tops.config and I have this error:
[Uploading 1.txt…]()

**Comment by Xiaohua-Eric-Xu** [comment]
@WY-SVG , As in previous threads, I think you need to actually upload a log file. The link you attached only redirects to this issue webpage.
